### PR TITLE
OpEx: Stale Cases Report

### PR DESCRIPTION
**Pull request by jimlerza**
Outputs a CSV of cases that are not closed or on appeal but haven't had a document filed in over a year.

Unfortunately, I was unable to achieve this without querying for the "most recently filed document" in each case individually.

I tried to do a `terms` query on `docketNumber.S`, providing the not-closed-not-on-appeal docket numbers (roughly 21,000) as the input. According to the OpenSearch documentation, terms queries have a limit of 65,536 terms, so I'm not quite sure why this didn't work.

Undeterred, I was pleased with the performance gains when executing the "get most recently filed document" queries in a priority queue:

Execution time with a `for of` loop: 13 minutes 12 seconds
Execution time with `pqueue`: 34 seconds

Just imagine how much easier these kinds of reports will be when all our data is in postgres!

### CODE REVIEW

File: scripts/reports/stale-cases.ts [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  aCase: RawCase;
+  applicationContext: ServerApplicationContext;
+}): Promise<void> => {
+  const mostRecentDocketEntry = await getMostRecentDocketEntry({
+    applicationContext,
+    docketNumber: aCase.docketNumber,
+  });
+  const deRcvdAt = mostRecentDocketEntry?.receivedAt;
+  const deAge = deRcvdAt ? calculateDifferenceInDays(todayISO, deRcvdAt) : 0;
+  if (deAge > 364) {
```
Comment (jtdevos) [inline scripts/reports/stale-cases.ts]: You have awakened the *Constant Curmudgeon*, who will provide little substantive feedback and will instead whine about replacing magic numbers with constants.

**Review by jtdevos — CHANGES_REQUESTED** [review]
Looks good!  Just one tweak requested.

File: scripts/reports/stale-cases.ts (hunk 1 above)
Comment (Mwindo) [reply scripts/reports/stale-cases.ts]: > You have awakened the Constant Curmudgeon, who will provide little substantive feedback and will instead whine about replacing magic numbers with constants.

This feels like it should be the text in a card game.

**Review by Mwindo — APPROVED** [review]

File: scripts/reports/stale-cases.ts (hunk 1 above)
Comment (jimlerza) [reply scripts/reports/stale-cases.ts]: Ha! Your suggestion is appreciated. :)

**Review by jtdevos — APPROVED** [review]

**Pull request merged**